Source/WebGPU/WebGPU/USDModel.swift

```diff
@@ -1049,146 +1154,209 @@ extension DDBridgeReceiver {
     }
 }
 
-final class Converter {
-    static func toWebImageAsset(_ asset: LowLevelTexture.Descriptor, data: Data) -> DDBridgeImageAsset {
-        DDBridgeImageAsset(
-            data: data,
-            width: asset.width,
-            height: asset.height,
-            depth: asset.depth,
-            bytesPerPixel: 0, // client calculates this
-            textureType: asset.textureType,
-            pixelFormat: asset.pixelFormat,
-            mipmapLevelCount: asset.mipmapLevelCount,
-            arrayLength: asset.arrayLength,
-            textureUsage: asset.textureUsage,
-            swizzle: asset.swizzle
+nonisolated func toWebImageAsset(_ asset: LowLevelTexture.Descriptor, data: Data) -> DDBridgeImageAsset {
+    DDBridgeImageAsset(
+        data: data,
+        width: asset.width,
+        height: asset.height,
+        depth: asset.depth,
+        bytesPerPixel: 0, // client calculates this
+        textureType: asset.textureType,
+        pixelFormat: asset.pixelFormat,
+        mipmapLevelCount: asset.mipmapLevelCount,
+        arrayLength: asset.arrayLength,
+        textureUsage: asset.textureUsage,
+        swizzle: asset.swizzle
+    )
+}
+
+nonisolated func convertSemantic(_ semantic: LowLevelMesh.VertexSemantic) -> Int {
+    switch semantic {
+    case .position: return 0
+    case .color: return 1
+    case .normal: return 2
+    case .tangent: return 3
+    case .bitangent: return 4
+    case .uv0: return 5
+    case .uv1: return 6
+    case .uv2: return 7
+    case .uv3: return 8
+    case .uv4: return 9
+    case .uv5: return 10
+    case .uv6: return 11
+    case .uv7: return 12
+    default: return 13
+    }
+}
+
+nonisolated func webAttributesFromAttributes(_ attributes: [LowLevelMesh.Attribute]) -> [DDBridgeVertexAttributeFormat] {
```

Comment:
why non isolated?

Reply:
oh good question, its because I didn't realize I could just make it `fileprivate` - will remove the `nonisolated`